api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -207,12 +231,13 @@
     Iterable<String> getPropertyNames();
 
     /**
-     * Return all of the currently registered {@linkplain ConfigSource configuration sources} for this configuration.
-     * The returned sources will be sorted by descending ordinal value and name, which can be iterated in a thread-safe manner.
+     * Return all of the currently registered {@linkplain ConfigValueSource configuration sources} for this
+     * configuration. The returned sources will be sorted by descending ordinal value and name, which can be iterated
+     * in a thread-safe manner.
      *
      * @return the configuration sources
      */
-    Iterable<ConfigSource> getConfigSources();
+    Iterable<ConfigValueSource> getConfigSources();
```

Comment:
You should not delete the previous one. I think you should add the new method `getConfigValueSources()`

Reply:
Ok!